### ISSUE TITLE
git checkout filename too long

### ISSUE THREAD

**Issue by ttrungvo** [audreyr/cookiecutter#398]
Hi guys, 

As you know, we can template a directory name and file like {{cookiecutter.dir_name}} and {{cookiecutter.file_name}}.txt.

With that convention, the file name (including the path) C://Templates/{{cookiecutter.dir_name}}/...../{{cookiecutter.file_name}}.txt could easily over 256 characters and git can't checkout a file name that are longer than 256 characters (http://stackoverflow.com/questions/22575662/filename-too-long-in-git-for-windows)

- is there anyway, we can alias cookiecutter to cc so we could use something like {{cc.dir_name}} and {{cc.file_name}}
- The other option is turning on core.longpaths so msysgit could support long file name (however, this might lead to some git scripts not reliable)

Any thoughts/suggestions?
Cheers,
Tony

**Comment by pydanny** [comment]
@ttrungvo - This is similar to requests of others to truncate the name of `cookiecutter` to something smaller such as `cc` or `bake`. Unfortunately, other projects have taken those abbreviations. Therefore, the best course of action is to alias the name `cookiecutter` to something that works on your own system.

**Comment by ttrungvo** [comment]
@pydanny how do I alias the name cookiecutter to something like cct and have the cookiecutter apply value to those cookie template variables? is there any wiki, example for me to follow?

thank you for your prompt response.

Cheers,
Tony

**Comment by ttrungvo** [comment]
@pydanny, I looked around the issue and I found these 2 issues that talked about alias: "Add aliases for cookiecutters #246" and "Allow user-defined aliases for cookiecutters #270", but that's the settings that needs to be set in the cookiecutterrc file.

The main problem that I'm facing is that on our build machine, it couldn't checkout the file name like "....{{cookiecutter.dir_name}}/...../{{cookiecutter.file_name}}.txt" because the file name is longer than 256 characters.

So maybe the question I should have asked is that is there any default alias for cookiecutter? or is there anyway to make the template variable {{cookiecutter.product_name}} shorter? (preferably without configure the cookierrc file?)

Cheers,
Tony
